cer-graphql/tests/integration-tests.test.js

```diff
@@ -13,6 +16,65 @@ async function createServerAndTestClient() {
     return createTestClient(new ApolloServer({ ...server, context: () => { } }));
 }
 
+/**
+ * Creates as testing server using the real server's configuration 
+ * with an injected context so we can add an authorization header to our query requests.
+ */
+async function createServerAndTestClientWithAuth() {
+    let server = await createServer(getCredentials(true));
+    let tokens = await getTokens();
+    return createTestClient(new ApolloServer({
+        ...server.config,
+        context: () => server.config.context({
+            req: {
+                headers: {
+                    authorization: `Bearer ${tokens['access_token']}`
+                }
+            }
+        })
+    }));
+}
+
+/**
+ * Gets OAuth tokens
+ */
+const getTokens = async () => {
+    let awsCreds;
+    try {
+        awsCreds = new aws.SharedIniFileCredentials({
+            profile: 'saml',
+        });
+    } catch (error) {
+        console.log("Could not retrieve AWS credentials. Try re-running the credential python script.");
+    }
+
+    let opts = {
+        host: 'ef54vsv71a.execute-api.ap-southeast-2.amazonaws.com',
```

Comment:
I'm not even sure what this is 😕 haha. But is there any chance we can at least move it to an `env` file?